app/assets/stylesheets/modules/tests-by-category-widget.styl

```diff
@@ -0,0 +1,32 @@
+.tests-by-category-table
```

Comment:
Avoid creating several top-level styles for the same widget:

```
.tests-by-category-table
  ...
.tests-by-category-td
  ...
.tests-by-category-progress
  ...
```

Add a class or ID to the widget, and scope the other classes underneath. Also, prefer semantic descriptions to direct references to HTML tags:

```
.tests-by-category-widget
  .categories-list
    ...
  .description
    ...
  .progress
    ...
```